api/api/controllers/task_controller.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import logging
+
+from aiohttp import web
+
+from api.encoder import dumps, prettify
+from api.util import remove_nones_to_dict, parse_api_param, raise_if_exc
+from wazuh.core.cluster.dapi.dapi import DistributedAPI
+from wazuh.core.common import database_limit
+from wazuh.task import get_task_status
+
+logger = logging.getLogger('wazuh')
+
+
+async def get_tasks_status(request, pretty=False, wait_for_complete=False, offset=0, limit=database_limit,
+                           tasks_list=None, agents_list=None, command=None, node=None, module=None, status=None, q=None,
+                           search=None, select=None, sort=None):
+    """Check the status of the specified tasks
+
+    Parameters
+    ----------
+    request
```

Comment:
Remove empty `request` parameter from docstring. It is not present in any other controller definitions.